sdk/src/ingredient.rs

```diff
@@ -328,6 +333,11 @@ impl Ingredient {
         self.data.as_ref()
     }
 
+    /// Returns a reference to the ocsp responses if it exists.
+    pub fn oscp_responses_ref(&self) -> Option<&Vec<ResourceRef>> {
```

Comment:
Probably make this pub(crate) Ingredient is a public API, but I guess it doesn't do much harm